portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/Forms.testcase

```diff
@@ -213,7 +213,11 @@ definition {
 
 		Navigator.openSiteURL(siteName = "Form Site");
 
-		FormsAdmin.addForm();
```

Comment:
@leticiachiu, please see this [comment](https://github.com/liferay-forms/liferay-portal/pull/894#discussion_r703070100) and think if we can apply the same suggestion here. :wink: